loadouts/faction-list.sqf

```diff
@@ -132,6 +132,10 @@ switch (toUpper(_faction)) do {
     case "FRWINTER": {
         #include "factions\frwinter.sqf"
     };
+    
+    case "FRWOOD": {
+        #include "factions\frwinter.sqf"
```

Comment:
Should be `#include "factions\frwood.sqf"`

Reply:
oh yeah nice, i haven't seen that thanks